commands/playwright-keplr.js

```diff
@@ -194,6 +194,19 @@ module.exports = {
     }
     return element;
   },
+  async waitForElementsByText(text, page = keplrWindow, exact = false) {
+    await module.exports.waitUntilStable(page);
+    const elements = await page.getByText(text, { exact: exact }).all();
+    await Promise.all(elements.map(element => element.waitFor()));
+    if (process.env.STABLE_MODE) {
```

Comment:
Why do we need this: `await Promise.all(elements.map(element => element.waitFor()));`?